### PR TITLE
Listing index view

### PR DESCRIPTION
**Pull request by dylancherry98**
Hi Team,

I designed a rough version of the views page for the 'Listings' index.

Please let me know what you think!

image

### CODE REVIEW

File: app/assets/stylesheets/components/_card.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.card-listing {
+  overflow: hidden;
+  background: white;
+  box-shadow: 0 0 15px rgba(0,0,0,0.2);
+  margin: 40px;
+  border-radius: 8px;
+}
+
+.card-listing > img {
```
Comment (MmeIsabelle) [inline app/assets/stylesheets/components/_card.scss:9]: Lines 9 to 41 can be nested inside your .card-listing. See the example.
image

File: app/assets/stylesheets/components/_index.scss [hunk 2]
```diff
@@ -1,4 +1,16 @@
 // Import your components CSS files here.
 @import "alert";
 @import "avatar";
+@import "card";
 @import "navbar";
+
+h1 {
```
Comment (MmeIsabelle) [inline app/assets/stylesheets/components/_index.scss:7]: You want to change the h1 style for the whole site?

File: app/views/listings/index.html.erb [hunk 3]
```diff
@@ -1,6 +1,16 @@
...
+<div class="cards">
+  <% @listings.each_with_index do |listing, index|  %>
+      <div class="card-listing">
+        <%= image_tag listing.photo %>
+        <div class="card-listing-infos">
+          <div>
+            <h2><%= listing.name %></h2>
+            <p><%= listing.description %></p>
+          </div>
+          <h2 class="card-listing-pricing"><%= number_to_currency(listing.price) %> / day.</h2>
```
Comment (MmeIsabelle) [inline app/views/listings/index.html.erb:11]: Remove the period after day.
Some cards show the price on 2 lines and it is not as nice.

File: app/views/listings/index.html.erb [hunk 4]
```diff
@@ -1,6 +1,16 @@
-<h1>Listing Index Page</h1>
-<ul>
-  <% @listings.each do |listing|  %>
-    <li><%= listing.name %></li>
+<h1>Listings:</h1>
```
Comment (MmeIsabelle) [inline app/views/listings/index.html.erb:1]: Remove the : after Listings.

**Comment by MmeIsabelle** [comment]
@dylancherry98 You can review my comments in the code.

**Review by MmeIsabelle — CHANGES_REQUESTED** [review]
And make your card clickable so we can access the show page. The view is not created yet, but use the listing_path

File: app/assets/stylesheets/components/_index.scss (hunk 2 above)
Comment (dylancherry98) [reply app/assets/stylesheets/components/_index.scss:7]: Fixed in my latest push!

**Comment by dylancherry98** [comment]
Hi Team,

Please don't push yet, I am now receiving an error when going to the listings index page.
@MmeIsabelle @Tien9320 @nafriedman

**Pull request merged**